### ISSUE TITLE
Where is the Machine Learning?

### ISSUE THREAD

**Issue by marcelluzs** [datasciencebr/serenata-de-amor#61]

**Comment by Irio** [comment]
Hi Marcellus,

As you can see in the issue tracker, there are many people working on tasks, directly related or not with Machine Learning. Data collection and analyses to find cases and get insights, for instance. Just with this there's already plenty of work to do (dozens of receipts getting a manual investigation before report to legal authorities), such as delegating to investigators and defining a process that will generate new datasets for Supervised Learning.

In summary, no ML is currently published in the repository. If you want to contribute on that end, are more than welcome. When the times comes I'll personally put my efforts on this.

**Comment by marcelluzs** [comment]
Please, don't hesitate in contacting me when you are ready to deploy some classification algorithms.

**Comment by ryukinix** [comment]
I'm curious about this part of the project too. Great ideal, btw.
